### DISCUSSION TITLE
Keyboard on screen

### DISCUSSION THREAD

**Discussion by ZWORX52** [monkeytypegame/monkeytype#2249] [Ideas] [answered]
I would like there to be a feature for the Simon Says keyboard to be on screen for a normal typing test. I like this keyboard and think that it adds something that looks nice. I was debating for this to be either a funbox or a normal setting, my reasoning for this being a funbox is it would spoil the fun of read aheads as a setting. It is really simple to implement (I hope) as all you need to do is have a funbox but the CSS file looks like this `#words .word{}`. I just took the Simon Says funbox and client-modified the CSS with chrome developer tools.

Thanks for your consideration, and please yell at me in full caps if this is already a feature.

Hope you stop feeling so pressured, Miodec! It's really an amazing site <3

**Comment by Miodec** [comment, marked as answer]
In the settings or commandline look for `keymap`. Simon says mode uses `keymap mode next`
